extensions/PhabBugz/lib/User.pm

```diff
@@ -131,13 +131,17 @@ sub match {
         attachments => { 'external-accounts' => 1 }
     };
 
+    # We can only fetch 100 users at a time so we need to do this in lumps
     my $phab_users = [];
-    my $result = request( 'user.search', $data );
-
-    if ( exists $result->{result}{data} && @{ $result->{result}{data} } ) {
-        foreach my $user ( @{ $result->{result}{data} } ) {
-            push @$phab_users, $class->new($user);
+    while (1) {
+        my $result = request( 'user.search', $data );
+        if ( exists $result->{result}{data} && @{ $result->{result}{data} } ) {
+            foreach my $user ( @{ $result->{result}{data} } ) {
+                push @$phab_users, $class->new($user);
+            }
         }
+        last if !$result->{cursur}->{after};
```

Comment:
cursur is misspelled, according to the doc you linked to it is cursor.

Rather than a while loop with 'last', I'd prefer seeing a do { } while $result->{cursor}{after}